shared/profile/user/weboftrust/index.tsx

```diff
@@ -19,7 +19,7 @@ type Props = {
   vouchedAt: number
 }
 
-const entry = (props: Props) => (
+const entry = (props: Props, statusStatement: string) => (
```

Comment:
I took a look at the git blame here, wondering why it was changed from `const WebOfTrust` to `const entries = (...) => ( ... )`

See comment below.